### PR TITLE
Move CI image to a supported version

### PR DESCRIPTION
**Pull request by telphan**
The latest ubuntu version has removed support for Erlang/OTP 23.

We should probably upgrade the min Elixir/Erlang version for this library, but as this issue is blocking every PR, we will consider that work later.

### CODE REVIEW

File: .github/workflows/static-analysis.yml
Comment (nlopes) [inline .github/workflows/static-analysis.yml:1]:
```suggestion
name: Static Analysis
```

**Review by nlopes — APPROVED** [review]
Approving as this seems to be blocking but once we unblock, let's work on fixing the issue - 18.04 has been post EOL for a few months now.

File: .github/workflows/static-analysis.yml
Comment (nlopes) [reply .github/workflows/static-analysis.yml:1]: ☝️ needs to be fixed before merging so please don't do auto-merge before fixing!

**Comment by telphan** [comment]
@nlopes I moved it to LTS 20.04, as 18.04 is no longer supported by GH.

**Pull request merged**
